model/mediafile.go

```diff
@@ -159,6 +162,9 @@ func (mfs MediaFiles) ToAlbum() Album {
 	a.Genre = slice.MostFrequent(a.Genres).Name
 	slices.SortFunc(a.Genres, func(a, b Genre) bool { return a.ID < b.ID })
 	a.Genres = slices.Compact(a.Genres)
+	a.Publisher = slice.MostFrequent(a.Publishers).Name
```

Comment:
Maybe I'm nitpicking, but wouldn't we want `a.Publisher` to be the one from the oldest album (i.e. earliest release date), not the most frequent?

Reply:
fixed